examples/stage-2-shield-study/addon/bootstrap.js

```diff
@@ -126,6 +131,7 @@ async function chooseVariation() {
   return toSet;
 }
 
+// TODO marcrowo: Move to lib?
```

Comment:
In the end it's up to @gregglind, but I think that this abstraction on top of the Cu.import system is actively harmful. It confuses people and static analysis, is something that will make other developers confused.

There is something to be sad about automatically unloading modules though, as that is important. I don' think this is the right solution though.